test/box-luatest/gh_6635_garbage_of_last_prep_txn_test.lua

```diff
@@ -0,0 +1,43 @@
+local server = require('test.luatest_helpers.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.before_all = function()
+    g.server = server:new{
+        alias   = 'default',
+        box_cfg = {memtx_use_mvcc_engine = true}
+    }
+    g.server:start()
+end
+
+g.after_all = function()
+    g.server:drop()
+end
+
+g.test_mvcc_txm_story_gc_does_not_break_memtx_tree_iter = function()
+    g.server:exec(function()
+        local t = require('luatest')
+
+        local s = box.schema.create_space('test')
+        s:create_index('pk')
+
+        -- Collect garbage before replacing tuples
+        box.internal.memtx_tx_gc(10)
```

Comment:
Why is `10` enough?

Reply:
Because we don't have many tuples at the moment.